test/DebugInfo/guard-let.swift

```diff
@@ -8,12 +8,9 @@ func use<T>(_ t: T) {}
 public func f(_ i : Int?)
 {
   // CHECK: define {{.*}}@"$S4main1fyySiSgF"
-  // The shadow copy store should not have a location.
-  // Note that the store must be in the same scope or else it might defeat
-  // livedebugvalues.
+  // CHECK1: %debug.copy = alloca %TSiSg
+  // CHECK1: @llvm.dbg.declare(metadata %TSiSg* %debug.copy
   // CHECK1: @llvm.dbg.declare(metadata {{(i32|i64)}}* %val.addr, {{.*}}, !dbg ![[DBG0:.*]]
-  // CHECK1: %[[PHI:.*]] = phi
-  // CHECK1: store {{(i32|i64)}} %[[PHI]], {{(i32|i64)}}* %val.addr, align {{(4|8)}}, !dbg ![[DBG1:.*]]
```

Comment:
Shouldn't we still store a shadow value for the phi somewhere? Can we keep checking for that?